### PR TITLE
blueprints/addon: Fix incorrect job formatting in `.travis.yml` config

### PR DESCRIPTION
**Pull request by buschtoens**
Fixes https://github.com/ember-cli/ember-cli/pull/8299/files#r246874738

### CODE REVIEW

**Comment by Turbo87** [comment]
@buschtoens can you target this at the release branch?

**Comment by buschtoens** [comment]
Sure. One sec, while I have a fight with the GitHub UI.

**Comment by buschtoens** [comment]
There we are 😊

File: blueprints/module-unification-addon/files/.travis.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ env:
   matrix:
     # we recommend new addons test the current and previous LTS
     # as well as latest stable release (bonus points to beta/canary)
-    - EMBER_TRY_SCENARIO=ember-lts-2.18
+      EMBER_TRY_SCENARIO=ember-lts-2.18
```
Comment (rwjblue) [inline blueprints/module-unification-addon/files/.travis.yml]: This change isn't correct. This one wasn't using stages yet, I think removing this `-` results in invalid yaml syntax.

**Pull request merged**